c10/core/DispatchKey.cpp

```diff
@@ -44,6 +44,8 @@ const char* toString(DispatchKey t) {
       return "AutocastTensorId";
     case DispatchKey::TESTING_ONLY_GenericWrapperTensorId:
       return "TESTING_ONLY_GenericWrapperTensorId";
+    case DispatchKey::Profiler:
+      return "ProfileId";
```

Comment:
Just return Profiler here. Also nb: https://github.com/pytorch/pytorch/pull/36240 is coming